templates/cassiopeia/scss/global/autumm.scss

```diff
@@ -0,0 +1,5 @@
+:root {
```

Comment:
You could import the ```_variables.scss``` before here and define the colors there

Reply:
We will see... I'm not sure yet how we should put all things together